Formula/mariadb.rb

```diff
@@ -151,6 +151,9 @@ def caveats; <<-EOS.undent
 
     To connect:
         mysql -uroot
+
+    Using brew services will place a plist file at #{ENV["HOME"]}/Library/LaunchAgents/#{plist_name}
```

Comment:
Sure but it's not very helpful to say go edit a plist file without noting where that plist file is or even how to find it. I see that running `brew services -v start mysql` tells you it's placing a plist file and where it's placed. This was not obvious to me to even try because I didn't know the plist file was overriding the bind address. I think adding some info to the caveat is the quickest/simplest solve to this problem and will save people a lot of time troubleshooting.

How about replacing this instead?
```
To connect run:
    mysql -uroot

The plist file for this service overrides the bind address to 127.0.0.1.
Edit this file if you need to change the server's bind address.
Enable verbose mode when starting via brew services for more detail on the plist file.
```